os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/inc/bt_board.h

```diff
@@ -79,7 +79,7 @@ extern "C" {
 #define EFUSE_SW_DRIVER_DEBUG_LOG    BIT6
 #define EFUSE_SW_RSVD2               BIT7
 
-#define CHECK_SW(x)                  (HAL_READ32(SPI_FLASH_BASE, FLASH_SYSTEM_DATA_ADDR + 0x28) & x)
+#define CHECK_SW(x)                  1 /*(HAL_READ32(SPI_FLASH_BASE, FLASH_SYSTEM_DATA_ADDR + 0x28) & x)*/
```

Comment:
Can we use any debug config check for BT ?
like
#ifdef bt_debug
#define check_sw(x) ..
#else
#define check_sw(x) ..

Reply:
Okay, I will add a debug config for it.
it will be depend on AMEBAD_BLE.